solutions/devsprint-cecilia-saraiva-2/FinanceAppSnapshotTests/ActivityViewCellSnapshotTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ActivityViewCellSnapshotTests.swift
+//  FinanceAppTests
+//
+//  Created by Raline Maria da Silva on 24/08/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import FinanceApp
+
+class AcitivityCellViewSnapshotTests: XCTestCase {
```

Comment:
[sugestao] alem de um typo, outra sugestao e adicionar `final`:
```suggestion
final class ActivityCellViewSnapshotTests: XCTestCase {
```

Reply:
[sugestao] Outra sugestao parecida e que podemos seguir a mesma estrutura de pastas do projeto original aqui tambem. So uma ideia, o que acha?

Reply:
feito